### ISSUE TITLE
Uncaught SyntaxError: Octal escape sequences are not allowed in strict mode

### ISSUE THREAD

**Issue by makingworld** [parcel-bundler/parcel#701]
**Choose one:** is this a 🐛 bug report or 🙋 feature request?
a 🐛 bug repor

### 🎛 Configuration (.babelrc, package.json, cli command)

```js
{
  "name": "webui",
  "description": "A Vue.js project",
  "version": "1.0.0",
  "author": "'romy' <'romy@gmail.com'>",
  "license": "MIT",
  "private": true,
  "scripts": {
    "dev": "parcel index.html",
    "build": "parcel build index.html",
    "clean": "rimraf .cache dist"
  },
  "browserslist": [
    "> 1%",
    "last 2 versions",
    "not ie <= 8"
  ],
  "devDependencies": {
    "babel-preset-env": "^1.6.0",
    "babel-preset-stage-3": "^6.24.1",
    "cross-env": "^5.0.5",
    "parcel-bundler": "parcel-bundler/parcel#vue",
    "rimraf": "^2.6.2",
    "vue": "^2.5.13",
    "vue-router": "^3.0.1",
    "vuex": "^3.0.1",
    "vuex-router-sync": "^5.0.0"
  }
}
```

### 🤔 Expected Behavior
running vue project

### 😯 Current Behavior
run failed

### 💁 Possible Solution
unknown

### 🔦 Context
 I just use shawwn/parcel-simple vue templete and didn't change any code

### 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           |1.4.1
| Node             |8.6.0
| npm/Yarn         |5.3.0
| Operating System | windows10

**Comment by DeMoorJasper** [comment]
@shawwn could u have a look at this?

**Comment by jpsouzasilva** [comment]
This somehow broke for me too! I was able to run it fine just the other day, then in the following week I could not get it to work with the inbuilt presets.

**What I had to change to make it work**

`"presets": [["env", { "modules": false }], "stage-3"],`

in the .babelrc file to:

`"presets": [
    "es2017",
    "es2016",
    "babel-preset-es2015-without-strict",
    "stage-3"
  ],`

So basically change the 2015 preset to allow strict mode during compilation and specify each other preset individually.

By the way, like I said, it somehow broke randomly. Preset `env` was working for me just the other day.
